src/app/renderer.js

```diff
@@ -34,13 +34,14 @@ function Renderer (editor, updateFiles, udapp, executionContext, formalVerificat
   })
 }
 
-Renderer.prototype.error = function (message, container, noAnnotations, type) {
+Renderer.prototype.error = function (message, container, options) {
   var self = this
-  if (!type) {
-    type = utils.errortype(message)
+  var opt = options || {}
+  if (!opt.type) {
+    opt.type = utils.errortype(message)
   }
-  var $pre = $('<pre />').text(message)
-  var $error = $('<div class="sol ' + type + '"><div class="close"><i class="fa fa-close"></i></div></div>').prepend($pre)
+  var $pre = $(opt.useSpan ? '<span />' : '<pre />').text(message)
```

Comment:
Yes, I noticed that, too. But then you do want the browser to break. Perhaps we can go with html and use `<br/>` for newlines?